cmake/Findmkl.cmake

```diff
@@ -0,0 +1,104 @@
+if (MKL_INCLUDE_DIRS AND MKL_LIBRARIES AND MKL_INTERFACE_LIBRARY AND
```

Comment:
add a header (block of comments) which explains the variables used by this Findmkl module and which variables are set